lib/lotus/repository.rb

```diff
@@ -296,10 +249,14 @@ def persist(entity)
       #   created_article = ArticleRepository.create(existing_article) # => no-op
       #   created_article # => nil
       #
+      #   imported_article = ArticleRepository.create(Article.new(title: 'New Lotus features', id: 123))
+      #   imported_article.id  # => 123
       def create(entity)
-        unless _persisted?(entity)
-          _touch(entity)
+        _touch(entity)
+        begin
           @adapter.create(collection, entity)
+        rescue Exception => e
+          raise Lotus::Model::AlreadyPersistedEntityError.new e.message
```

Comment:
Yeah, @pascalbetz is right:

1. [Never catch `Exception`](http://stackoverflow.com/questions/10048173/why-is-it-bad-style-to-rescue-exception-e-in-ruby). 
2. There are exceptions that we want to let to bubble, for instance a database integrity violation.
3. It's too generic to assume that all the exceptions are raised because the entity was already persisted.